packages/web-components/src/accordion/accordion.template.ts

```diff
@@ -7,7 +7,7 @@ import type { Accordion } from './accordion.js';
 export function accordionTemplate<T extends Accordion>(): ElementViewTemplate<T> {
   return html<T>`
     <template>
-      <slot ${slotted({ property: 'slottedAccordionItems', filter: elements() })}></slot>
+      <slot @slotchange="${(x, c) => x.slotchangeHandler(c.event as Event)}"></slot>
```

Comment:
Do we know why slotted is not working specifically in WebKit?

Reply:
I don't - the same thing seems to happen with the `children()` directive. For example, our tests do this:

```ts
await page.setContent(/* html */ `
  <fluent-accordion expand-mode="multi">
    <fluent-accordion-item>
      <span slot="heading">Heading 1</span>
      <div>Content 1</div>
    </fluent-accordion-item>
    <fluent-accordion-item>
      <span slot="heading">Heading 2</span>
      <div>Content 2</div>
    </fluent-accordion-item>
  </fluent-accordion>
`);
```

Under the hood, Playwright's `page.setContent()` operates similarly to `document.write()`.

Reply:
Hmmm...I'm going to pull this down this afternoon and look into this. Not blocking, but it would be good to know if there's a bug in FAST or in Webkit happening somewhere to get resolution on the root cause.